includes/admin/class-wcmp-admin.php

```diff
@@ -539,6 +550,89 @@ public static function getTrackTraceUrl($order_id, $track_trace)
         return $trackTraceUrl;
     }
 
+    /**
+     *
+     */
+    public function productHsCodeField()
+    {
+        echo '<div class="options_group">';
+        woocommerce_wp_text_input(
+            [
+                'id' => self::META_HS_CODE,
+                'label' => __('HS Code', 'woocommerce-myparcelbe'),
+                'description' => sprintf(
+                    __('HS Codes are used for MyParcel world shipments, you can find the appropriate code on the %ssite of the Belgium Customs%s.',
+                        'woocommerce-myparcelbe' , '<a href="http://tarief.douane.nl/arctictariff-public-web/#!/home" target="_blank">',
+                        '</a>'
+                    )
+                ),
+            ]
+        );
+        echo '</div>';
+    }
+
+    /**
+     * @param int $post_id
+     */
+    public function productHsCodeFieldSave(int $post_id)
+    {
+        // check if hs code is passed and not an array (=variation hs code)
+        if (isset($_POST[self::META_HS_CODE]) && ! is_array($_POST[self::META_HS_CODE])) {
+            $product = wc_get_product($post_id);
+            $hs_code = $_POST[self::META_HS_CODE];
+            if (! empty($hs_code)) {
+                WCX_Product::update_meta_data($product, self::META_HS_CODE, esc_attr($hs_code));
+
+                return;
+            }
+            if (isset($_POST[self::META_HS_CODE]) && empty($hs_code)) {
+                WCX_Product::delete_meta_data($product, self::META_HS_CODE);
+
+                return;
+            }
+        }
+
+        return;
+    }
+
+    /**
+     *
```

Comment:
Deze dockblock is leeg, je kan hier altijd neerzetten wat er in de method gebeurd